packages/mermaid/src/diagrams/flowchart/flowDb.ts

```diff
@@ -255,24 +251,24 @@ export const setDirection = function (dir: string) {
  * @param className - Class to add
  */
 export const setClass = function (ids: string, className: string) {
-  ids.split(',').forEach(function (_id) {
-    const id = _id;
-    if (vertices[id] !== undefined) {
+  for (const id of ids.split(',')) {
+    if (vertices[id]) {
       vertices[id].classes.push(className);
     }
-
-    if (subGraphLookup[id] !== undefined) {
+    if (subGraphLookup[id]) {
       subGraphLookup[id].classes.push(className);
     }
-  });
+  }
 };
 
 const setTooltip = function (ids: string, tooltip: string) {
-  ids.split(',').forEach(function (id) {
-    if (tooltip !== undefined) {
-      tooltips[version === 'gen-1' ? lookUpDomId(id) : id] = sanitizeText(tooltip);
-    }
-  });
+  if (!tooltip) {
```

Comment:
There's a potential change in behaviour if `tooltip` is `""` (empty string). If we want to avoid this change, we can do the following:

```suggestion
  if (tooltip === undefined) {
```

But, the typescript type for `tooltip` is `string`, so maybe TypeScript will confirm that this string is never `undefined`? I don't know if we're using `any` for this type anywhere.